modules/external-secrets/data.tf

```diff
@@ -1,10 +1,7 @@
 data "aws_iam_policy_document" "external_secrets" {
   statement {
     actions = ["ssm:GetParameter"]
-    resources = concat(
-      var.external_secrets_ssm_parameter_arns,
-      ["arn:${var.addon_context.aws_partition_id}:ssm:${var.addon_context.aws_region_name}:${var.addon_context.aws_caller_identity_account_id}:parameter/*"]
```

Comment:
This was previously hard-coding `:parameter/*` that could not be configured using `var.external_secrets_ssm_parameter_arns`, allowing wide-open access.

Reply:
@rodrigobersa you added this originally as part of https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1367 - perhaps you could provide some context as to why it was added, and whether it can/should be removed.  Thanks!